list_shiptypes.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+import yaml
+import json
+
+if len(sys.argv) < 2:
+    print("Usage: python3 convert.py <path/to/eve-sde/fsd>")
+    exit(1)
+
+path = sys.argv[1]
+
+os.makedirs("dist", exist_ok=True)
+
+with open(f"{path}/groupIDs.yaml") as fp:
+    groupIDs = yaml.load(fp, Loader=yaml.CSafeLoader)
+
+with open(f"{path}/typeIDs.yaml") as fp:
+    typeIDs = yaml.load(fp, Loader=yaml.CSafeLoader)
+
+ships = []
+
+for id, entry in typeIDs.items():
+    name = entry["name"]["en"]
+    groupID = entry["groupID"]
+    published = entry["published"]
+
+    group = groupIDs[entry["groupID"]]
+    groupName = group["name"]["en"]
+    categoryID = group["categoryID"]
+
+    if categoryID == 6 and published:
+        ships.append(
+            {
+                "id": id,
+                "name": name,
+                "group": groupName,
+            }
+        )
+
+with open("dist/ships.json", "w") as fp:
+    json.dump(ships, fp)
```

Comment:
This PR does much more than it suggests. Please split it up into two PRs, would also have made the review 1000 times easier. It is now a mix of adding new stuff content-wise, and changing infrastructure so an npm can be published. This makes it hard to distinguish what is what. Please don't :)